src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+  }
+
+  @for $index from 1 through 5 {
+    &--#{$index} &__star:nth-child(-n+#{$index}) {
+      background-image: url(../images/star-active.svg);
+
+    }
+  }
+
+& &__star:nth-child(5) {
+  background-image: url(../images/star.svg);
+}
```

Comment:
@KareemAchkar you seem to have a shared issue with your Brothers - `class="stars stars--5"` would not show me 5 active stars...